### PR TITLE
* bus events

### PR DESCRIPTION
**Pull request by am0nshi**
* bus events

### CODE REVIEW

File: Resources/config/services.yml [hunk 1]
```diff
@@ -26,6 +26,17 @@ services:
...
+    service.event.processor:
+        class: Garlic\Bus\Service\Processor\MulticastEventProcessor
+        public: true
+        arguments:
+            - "@service.request"
+            - "@service.response"
+            - "@router"
+            - "@kernel"
+        tags:
+            - { name: 'enqueue.client.processor', topicName: 'multicast_event'}
```
Comment (imaximius) [inline Resources/config/services.yml]: Can you exclude topicName to parrameter and add Namespace to them?

File: Service/Abstracts/RpcProducerAbstract.php [hunk 2]
```diff
@@ -89,4 +92,28 @@ public function sendCommand(string $service, $message, bool $reply = true)
...
+     */
+    public function sendEvent($name, $message)
+    {
+        if (false == $message instanceof Message) {
+            $message = $this->context->createMessage($message);
+        }
+
+        $message->setProperty(Config::PARAMETER_TOPIC_NAME, $name);
+
+        $topic = $this->context->createTopic('enqueue.default');
```
Comment (imaximius) [inline Service/Abstracts/RpcProducerAbstract.php:113]: Can you exclude to parameter?

File: Resources/config/services.yml (hunk 1 above)
Comment (am0nshi) [reply Resources/config/services.yml]: It's hardcoded value for inner usage, no need to fetch it into separate place.

**Review by demotivationme — APPROVED** [review]

File: Resources/config/services.yml (hunk 1 above)
Comment (am0nshi) [reply Resources/config/services.yml]: fixed

**Review by imaximius — APPROVED** [review]

**Pull request merged**